components/SingleClassView/Attendance/AttendanceRow.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import styles from "./attendance.module.css";
+import { Attendance } from "../../../models/Attendance";
+import { CustomError } from "../../util/CustomError";
+import { AttendanceTypes, CreateAttendance } from "../,,/../../../models";
+
+type AttendanceRowProps = {
+    onAttendanceChange: (userId: string, newAttendance: AttendanceTypes) => void;
+    name: string,
+    userId: string,
+    attendance: AttendanceTypes | undefined,
+};
+
+const AttendanceRow: React.FC<AttendanceRowProps> =  ({
+    onAttendanceChange,
+    name,
+    userId,
+    attendance,
+}) => {
+    console.log(attendance, name);
+    return (
+        <div className={styles.attendanceRow}>
+            <p className={styles.userName}>{name}</p>
+            <div className={styles.userAttendance}>
+                {attendance == "Present" ? (
+                    <button
+                        className={styles.presentButton}
+                    >Present</button>
+                ): (
+                    <button
+                        onClick={(_) => onAttendanceChange(userId, "Present")}
+                        className={styles.uncheckedButton}
```

Comment:
we can put the conditional operator in the className, so we dont have to repeat buttons